src/main/java/com/atlassian/db/replica/spi/jdbc/JdbcProtocol.java

```diff
@@ -0,0 +1,17 @@
+package com.atlassian.db.replica.spi.jdbc;
```

Comment:
Is it SPI or API?

Reply:
It's API since we do not require the client to implement/extend this.
Will move it to the `api` package along with the `JdbcUrl` class